src/test/regress/sql/mx_application_name.sql

```diff
@@ -0,0 +1,10 @@
+CREATE SCHEMA mx_app_name;
+
+SET application_name TO mx_app_name;
```

Comment:
as far as I can tell this does not reproduce the issue, so this test seems redundant

Reply:
Maybe we can set PGAPPNAME in pg_regress_multi.pl somehow

Reply:
I think the trick you used for pg_dump works fine

Reply:
nope, that didn't work either